### PR TITLE
DCOS-62619: globally style all anchor tags

### PR DESCRIPTION
**Pull request by mperrotti**
Before:
Screen Shot 2020-01-09 at 5 34 46 PM

After:
Screen Shot 2020-01-09 at 5 34 25 PM

### CODE REVIEW

File: packages/shared/styles/normalize.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { themeTextColorInteractive } from "../../design-tokens/build/js/designTokens";
+
+// TODO: follow-up with Jira ticket DCOS-62772 (a more thorough set of normalize styles)
+export const normalize = `
+    a {
+        text-decoration: none;
```
Comment (juliangieseke) [inline packages/shared/styles/normalize.ts]: I think I'd keep this underlined in normalize file 🤔 we're removing _all_ underlines otherwise.

Comment (GeorgiSTodorov) [reply packages/shared/styles/normalize.ts]: @juliangieseke , aren't we just removing all underlines for link elements?

Comment (juliangieseke) [reply packages/shared/styles/normalize.ts]: yeah, thats whats happening here. My questions was if thats actually what we want?

Comment (GeorgiSTodorov) [reply packages/shared/styles/normalize.ts]: I misunderstood your comment as saying that we are removing all underlines for all elements. I think removing it for all link elements is what we want, but @mperrotti can answer better.

Comment (mperrotti) [reply packages/shared/styles/normalize.ts]: Removing the underline for all links is intentional

**Pull request merged**